src/main/java/com/wcc/platform/domain/platform/Partner.java

```diff
@@ -20,8 +19,8 @@
 @NoArgsConstructor
 @Builder
 public class Partner {
-  @NotEmpty private List<Image> images;
+  @NotEmpty private Image image;
```

Comment:
```suggestion
  @NotNull private Image image; 
```
I guess empty is for a list or string.

Reply:
Done.